[2022-05-06 11:22:22,197] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 manual__2022-05-06T05:52:14.709011+00:00 [queued]>
[2022-05-06 11:22:22,201] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 manual__2022-05-06T05:52:14.709011+00:00 [queued]>
[2022-05-06 11:22:22,201] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-06 11:22:22,201] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-05-06 11:22:22,201] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-06 11:22:22,208] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_2> on 2022-05-06 05:52:14.709011+00:00
[2022-05-06 11:22:22,210] {standard_task_runner.py:52} INFO - Started process 1465715 to run task
[2022-05-06 11:22:22,211] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', 'manual__2022-05-06T05:52:14.709011+00:00', '--job-id', '44', '--raw', '--subdir', '/home/sanjeev/Documents/SubDAGs/dags/paralle_processing.py', '--cfg-path', '/tmp/tmpw_j1534k', '--error-file', '/tmp/tmpu05y4724']
[2022-05-06 11:22:22,212] {standard_task_runner.py:77} INFO - Job 44: Subtask task_2
[2022-05-06 11:22:22,237] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_2 manual__2022-05-06T05:52:14.709011+00:00 [running]> on host kumar
[2022-05-06 11:22:22,269] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2022-05-06T05:52:14.709011+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-06T05:52:14.709011+00:00
[2022-05-06 11:22:22,269] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-06 11:22:22,270] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 4']
[2022-05-06 11:22:22,273] {subprocess.py:85} INFO - Output:
[2022-05-06 11:22:26,274] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-06 11:22:26,310] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20220506T055214, start_date=20220506T055222, end_date=20220506T055226
[2022-05-06 11:22:26,360] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-06 11:22:26,387] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
